demo/index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en-GB">
+  <head>
+    <meta charset="utf-8" />
+    <style>
+      body {
+        background: #fafafa;
+      }
+    </style>
+  </head>
+  <body>
+    <div id="demo"></div>
+
+    <script type="module">
+      import { html, render } from "lit";
+      import "../stacked-sheets.js";
+
+      const title = "Hello owc World!";
+      render(
+        html`
+          <stacked-sheets .title=${title}> some light-dom </stacked-sheets>
```

Comment:
It would be good if the demo page could also show use of the `stacked-sheets-holder` element. 
I don't recall if `stacked-sheets` works without the holder - assuming it does then good to show it both inside and, outside, a holder.

Reply:
It is using `stacked-sheets-holder`, the demo is just stuffing the app in a wrapper.

https://github.com/AqoviaElements/stacked-sheets/blob/86d9469bfc3100f89efb3e15848f16453828a335/src/my-app.js#L77